google/src/main/scala/org/broadinstitute/dsde/rawls/google/GoogleUtilities.scala

```diff
@@ -21,31 +22,45 @@ import scala.util.{Failure, Success, Try}
 trait GoogleUtilities extends LazyLogging with InstrumentedRetry with GoogleInstrumented {
   implicit val executionContext: ExecutionContext
 
-  protected def when500orGoogleError(throwable: Throwable): Boolean = {
+  protected def when500or400orGoogleError(throwable: Throwable): Boolean = {
```

Comment:
I renamed the existing `when500orGoogleError` to `when500or400orGoogleError` because the name was not accurate- it would retry both 5xx and 4xx status codes.

I ended up creating an actual `when500orGoogleError`, which does what it says